index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Database {
+    private $conn;
+
+    public function __construct() {
+        $this->conn = new mysqli("localhost", "root", "password", "my_database");
+
+        if ($this->conn->connect_error) {
+            die("Connection failed: " . $this->conn->connect_error);
+        }
+    }
+
+    public function saveData($name, $description, $price) {
+        $sql = "INSERT INTO items (name, description, price) VALUES ('$name', '$description', '$price')";
```

Comment:
Add input validation to ensure data integrity. 
```
if (empty($name) || empty($description) || !is_numeric($price)) {
        throw new Exception("Invalid input data");
}
```
Do not use raw SQL queries, vulnerable to SQL injection attacks. 
Use prepared statements:    
```
    $stmt = $this->conn->prepare("INSERT INTO items (name, description, price) VALUES (?, ?, ?)");
    $stmt->bind_param("ssd", $name, $description, $price);
    if (!$stmt->execute()) {
        echo "Error: " . $stmt->error;
    }
    $stmt->close();
```